bigchaindb/core.py

```diff
@@ -115,6 +117,17 @@ def info(self, request):
 
         self.abort_if_abci_chain_is_not_synced()
 
+        # Check if BigchainDB supports the Tendermint version
+        try:
+            if check_tendermint_version(request.version):
```

Comment:
This function (currently named `check_tendermint_version()`) returns a Boolean, so it could be renamed so that lines like this read more like plain English, e.g.

```python
if tendermint_version_is_compatible(request.version):
```

In other words, I'm suggesting renaming the function `check_tendermint_version()` as `tendermint_version_is_compatible()`.